app/js/controllers/settings.js

```diff
@@ -5,21 +5,39 @@
 export default class SettingsController extends Controller {
   constructor(options) {
     super(options);
+
+    this.merging = false;
+
+    this._refreshAddons = this._refreshAddons.bind(this);
```

Comment:
i don't think you can do that because you'd be creating a new function that's different than the one you previously added as an event listener